### ISSUE TITLE
template allows $ instead of @

### ISSUE THREAD

**Issue by kahsa** [vlang/v#14788]
**V version:** 
**OS:** 

**What did you do?**

https://github.com/vlang/v/blob/master/doc/docs.md#tmpl-for-embedding-and-parsing-v-template-files

I get the same result when I change `1.txt` as follows although this is intentional or a bug.

```
name: $name

age: $age

numbers: $numbers

@for number in numbers
  $number
@end
```

**What did you expect to see?**
The part where `@` is changed to `$` is ignored.
 
**What did you see instead?**
No error.

**Comment by mjh316** [comment]
I'm not familiar with this part of v, but I'm assuming in [here](https://github.com/vlang/v/blob/master/vlib/v/parser/tmpl.v#L76) is why the bug occurs; it turns '@' into '$', but if you already have $ initially, it'll just go straight to being processed.

**Comment by kahsa** [comment]
Hmm.. If V tries to parse `$`, there will be a conflict with the javascript template and  will not compile.

**Comment by felipensp** [comment]
I see it works just like is expected to work. Confirm @medvednikov ?

**Comment by medvednikov** [comment]
No, I think it should only allow @. One way and everything. I didn't intend $ to work, it's an unintended bug, since templates are compiled into V code, and $ is used for string interpolation.
